.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+  "prettier.semi": false,
+  "prettier.singleQuote": true
```

Comment:
Can the vscode plugin read from the `.prettierrc` instead of adding it here? Or maybe we could put this in the `package.json` to avoid a ton of dot files.

Reply:
Yeah, I can add `.vscode/` to the `.gitignore` if you'd like so this doesn't get committed again (I added this for redundancy).

In any case the vscode plugin looks at vscode settings *and* the .prettierrc!

Reply:
I think we may want things like `.vscode/launch.json` to be shareable, so I'm not against keeping `.vscode/` in git, but I'm wondering if the extension can automatically infer project settings from the dotfiles or package.json.

Maybe for now we just remove this `.vscode` settings file.